packages/devtools_app/lib/src/table.dart

```diff
@@ -719,6 +711,13 @@ class _TableState<T> extends State<_Table<T>> with AutoDisposeMixin {
     }
   }
 
+  void _initSearchListener() {
+    addAutoDisposeListener(
```

Comment:
nit: why not move the check that activeSearchMatchNotifier is not null into this method?

Reply:
done